content/software/wallets.md

```diff
@@ -32,5 +32,10 @@ interested in trying the new RGB v0.10 features please use the [command-line too
   working on RGB integration and RGB-related tools. Supports both fungible
   and non-fungible assets.
 
+* ### [Bitlight Wallet](https://bitlightlabs.com/wallet)
+
+  The first non-custodial Bitcoin wallet supporting the RGB protocol, currently
```

Comment:
```suggestion
  The first non-custodial Bitcoin wallet supporting the RGB protocol, currently
```

Can we please remove the part "the first"? Since there were many other RGB non-custodial wallets since 2015, part of which is listed above :)